qdrant-landing/content/documentation/cloud/qdrant-cloud-setup.md

```diff
@@ -1,108 +1,54 @@
 
-```bash
-curl \
-  -X GET 'https://xyz-example.eu-central.aws.cloud.qdrant.io:6333' \
-  --header 'api-key: <paste-your-api-key-here>'
-```
-Open Terminal and run the request. You should get a response that looks like this:
+Invited users will receive an email with an invitation link to join Qdrant Cloud. Once they signed up, they can accept the invitation from the Overview page.
 
-```bash
-{"title":"qdrant - vector search engine","version":"1.8.1"}
-```
+![Accepting invitation](/documentation/cloud/accept-invitation.png)
 
-> **Note:** You need to include the API key in the request header for every
-> request over REST or gRPC.
+## Switching between accounts
 
-## Authenticate via SDK
+If you have access to multiple accounts, you can switch between accounts with the account switcher on the top menu bar of the Qdrant Cloud Console.
 
-Now that you have created your first cluster and API key, you can access the
-Qdrant Cloud from within your application.
-Our official Qdrant clients for Python, TypeScript, Go, Rust, and .NET all
-support the API key parameter. 
+![Switching between accounts](/documentation/cloud/account-switcher.png)
 
-```python
-from qdrant_client import QdrantClient
+## Account settings
 
-qdrant_client = QdrantClient(
-    "xyz-example.eu-central.aws.cloud.qdrant.io",
-    api_key="<paste-your-api-key-here>",
-)
-```
+You can configure your account settings in the Qdrant Cloud Console, by clicking on your account picture in the top right corner, and selecting *Profile*.
 
-```typescript
-import { QdrantClient } from "@qdrant/js-client-rest";
+The following functionality is available.
 
-const client = new QdrantClient({
-  host: "xyz-example.eu-central.aws.cloud.qdrant.io",
-  apiKey: "<paste-your-api-key-here>",
-});
-```
+### Renamin an account
 
-```rust
-use qdrant_client::Qdrant;
+If you have use multiple accounts for different purposes, it is a good idea to give them descriptive names, for example *Development*, *Production*, *Testing*. You can also choose which account should be the default one, when you log in.
 
-let client = Qdrant::from_url("https://xyz-example.eu-central.aws.cloud.qdrant.io:6334")
-    .api_key("<paste-your-api-key-here>")
-    .build()?;
-```
+![Account management](/documentation/cloud/account-management.png)
 
-```java
-import io.qdrant.client.QdrantClient;
-import io.qdrant.client.QdrantGrpcClient;
+### Deleting an account
 
-QdrantClient client =
-    new QdrantClient(
-        QdrantGrpcClient.newBuilder(
-                "xyz-example.eu-central.aws.cloud.qdrant.io",
-                6334,
-                true)
-            .withApiKey("<paste-your-api-key-here>")
-            .build());
-```
-
-```csharp
-using Qdrant.Client;
-
-var client = new QdrantClient(
-  host: "xyz-example.eu-central.aws.cloud.qdrant.io",
-  https: true,
-  apiKey: "<paste-your-api-key-here>"
-);
-```
+When you delete an account, all database clusters and data associated with it will also be deleted.
```

Comment:
'All database clusters and associated data will be deleted'

Is a bit more concise